pyvista/core/pointset.py

```diff
@@ -891,15 +973,12 @@ def __sub__(self, cutting_mesh):
     @property
     def _offset_array(self):
         """Return the array used to store cell offsets."""
-        return _vtk.vtk_to_numpy(self.GetPolys().GetOffsetsArray())
+        return _get_offset_array(self.GetPolys())
```

Comment:
It looks like this `_offset_array` and `_connectivity_array` only are for faces.  I'm wondering if they should be `_faces_offset_array` and so on.  This comment is only really relevant if we will support strips or verts in this PR.